library/src/main/java/io/constructor/data/DataManager.kt

```diff
@@ -148,6 +148,32 @@ constructor(private val constructorApi: ConstructorApi, @ConstructorSdk private
         return constructorApi.getBrowseResultsCRT(dynamicUrl)
     }
 
+    fun getBrowseItemsResults(encodedParams: Array<Pair<String, String>> = arrayOf()): Observable<ConstructorData<BrowseResponse>> {
+        var dynamicUrl = "/${ApiPaths.URL_BROWSE_ITEMS}${getAdditionalParamsQueryString(encodedParams)}"
+        return constructorApi.getBrowseResults(dynamicUrl).map { result ->
```

Comment:
I just noticed that we're also using `getBrowseResults` here. It looks like it just takes a URL and the response is in the same format as a normal browse response. Doesn't seem like it would cause any issues? 🤔 

Should we create a separate `getBrowseItemResults` function in the `ConstructorApi` interface?